rigging/commands/destroy.py

```diff
@@ -12,8 +12,8 @@
 import os
```

Comment:
Can drop this import now.

Reply:
Ack

Reply:
Done b87891d8a1489e7e767367997dc5f212022e46e0 865eb1cd8db905ee4027ff327955c75b509805bb